helpers/cleanup.sh

```diff
@@ -14,12 +14,25 @@
 # limitations under the License.
 
 
-PROJECT_ID="$1"
+PROJECT_ID="$(gcloud projects list --format="value(projectId)" | grep -Eo "$1")"
+
+if [[ $PROJECT_ID == "" ]];
+then
+    echo "ERROR The specified project wasn't found."
+    exit 1;
+fi
+
 ORG_ID="$(gcloud projects describe ${PROJECT_ID} --flatten=parent.id | grep -Eo "\d+")"
-SERVICE_ACCOUNT_ID="$2"
-STAGING_DIR="$(echo ${PWD}/staging)"
+SERVICE_ACCOUNT_ID="$(gcloud iam service-accounts list --format="value(email)" | grep -Eo "$2@${PROJECT_ID}.iam.gserviceaccount.com")"
+KEY_FILE="$(echo ${PWD}/credentials.json)"
```

Comment:
I don't think echo is needed here. `KEY_FILE="${PWD}/credentials.json"` should do the same?

Reply:
Ah, you are correct.